.github/labeler-no-sync.yml

```diff
@@ -28,5 +28,38 @@
       - any-glob-to-any-file:
         - .github/workflows/*
         - ci/**/*.*
+  - all:
+    - head-branch:
+      # r-ryantm's branches have this prefix.
+      - ^auto-update/
+    - changed-files:
+      - any-glob-to-any-file:
+        # Browser updates should always be backported.
+        - pkgs/applications/kde/angelfish.nix
+        - pkgs/applications/kde/falkon.nix
+        - pkgs/applications/kde/konqueror.nix
+        - pkgs/applications/networking/browsers/**/*
+        - pkgs/by-name/br/brave/**/*
+        # Added in December 2024, uncomment for 25.05
+        # - pkgs/by-name/ca/catalyst-browser/**/*
+        - pkgs/by-name/ch/chawan/**/*
+        - pkgs/by-name/di/dillo-plus/**/*
+        - pkgs/by-name/di/dillo/**/*
+        - pkgs/by-name/ed/edbrowse/**/*
+        - pkgs/by-name/ep/epiphany/**/*
+        - pkgs/by-name/go/google-chrome/**/*
+        - pkgs/by-name/ly/lynx/**/*
+        - pkgs/by-name/mi/microsoft-edge/**/*
+        - pkgs/by-name/mi/midori-unwrapped/**/*
+        - pkgs/by-name/mu/mullvad-browser/**/*
+        - pkgs/by-name/op/opera/**/*
+        - pkgs/by-name/to/tor-browser/**/*
+        - pkgs/by-name/vi/vimb-unwrapped/**/*
+        - pkgs/by-name/w3/w3m/**/*
+        - pkgs/desktops/lomiri/applications/morph-browser/**/*
+        - pkgs/development/libraries/webkitgtk/**/*
+        - pkgs/kde/gear/angelfish/**/*
```

Comment:
The browser itself isn't security critical, qtwebengine is.
The same is true for others.